crates/sui-distributed-execution/src/exec_worker.rs

```diff
@@ -279,99 +279,99 @@ impl<
         );
     }
 
-    async fn _async_exec(
-        full_tx: TransactionWithEffects,
-        memory_store: Arc<S>,
-        child_inputs: HashSet<ObjectID>,
-        move_vm: Arc<MoveVM>,
-        reference_gas_price: u64,
-        epoch_id: EpochId,
-        epoch_start_timestamp: u64,
-        protocol_config: ProtocolConfig,
-        metrics: Arc<LimitsMetrics>,
-        my_id: u8,
-        ew_ids: &Vec<UniqueId>,
-    ) -> TransactionWithResults {
-        // DELETE THIS
-        // return TransactionWithResults {
-        //     full_tx: full_tx.clone(),
-        //     tx_effects: TransactionEffects::default(),
-        //     deleted: BTreeMap::new(),
-        //     written: BTreeMap::new(),
-        //     missing_objs: HashSet::new(),
-        // };
-
-        let tx = &full_tx.tx;
-        let txid = tx.digest();
-        let tx_data = tx.transaction_data();
-        let (kind, signer, gas) = tx_data.execution_parts();
-        let input_objects = Self::read_input_objects_from_store(memory_store.clone(), &tx).await;
-        let gas_status =
-            Self::get_gas_status(&tx, &input_objects, &protocol_config, reference_gas_price).await;
-        let shared_object_refs = input_objects.filter_shared_objects();
-        let transaction_dependencies = input_objects.transaction_dependencies();
-        let mut gas_charger = GasCharger::new(*tx.digest(), gas, gas_status, &protocol_config);
-        // println!(
-        //     "Dependencies for tx {}: {:?}",
-        //     txid, transaction_dependencies
-        // );
-        let temporary_store = TemporaryStore::new(
-            memory_store.clone(),
-            input_objects.clone(),
-            *txid,
-            &protocol_config,
-        );
-
-        let (inner_temp_store, tx_effects, _execution_error) =
-            execution_engine::execute_transaction_to_effects::<execution_mode::Normal>(
-                shared_object_refs,
-                temporary_store,
-                kind,
-                signer,
-                &mut gas_charger,
-                *txid,
-                transaction_dependencies,
-                &move_vm,
-                &epoch_id,
-                epoch_start_timestamp,
-                &protocol_config,
-                metrics.clone(),
-                false,
-                &HashSet::new(),
-            );
-
-        let mut missing_objs = HashSet::new();
-        let input_object_map = input_objects.into_object_map();
-        for read_obj_id in &inner_temp_store.runtime_read_objects {
-            if !input_object_map.contains_key(read_obj_id) && !child_inputs.contains(read_obj_id) {
-                missing_objs.insert(*read_obj_id);
-            }
-        }
+    // async fn _async_exec(
```

Comment:
dead code

Reply:
There is a lot, but I don't know if we want it for other testing purposes. @zablotchi could you do a cleanup of whatever is unnecessary?